book/templating.rst

```diff
@@ -1076,14 +1076,14 @@ configuration option.
 
 .. _`book-templating-version-by-asset`:
 
-If you need to set a version for a specific asset, you can set the ``version`` argument
+If you need to set a version for a specific asset, you can use the ``asset_version`` function
 if you are using Twig (or the fourth argument if you are using PHP) to the desired version:
 
 .. configuration-block::
 
     .. code-block:: html+jinja
 
-        <img src="{{ asset('images/logo.png', version='3.0') }}" alt="Symfony!" />
+        <img src="{{ asset_version('images/logo.png', '3.0') }}" alt="Symfony!" />
```

Comment:
Actually, you will now have to define the version in the asset configuration for the affected package and then call `asset_version()` with the package name (see http://symfony.com/blog/new-in-symfony-2-7-the-new-asset-component).